<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveformWrapper</class>
 <widget class="QWidget" name="WaveformWrapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="waveformGroupBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Waveform Wrapper</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="waveformLayout">
        <item>
         <layout class="QHBoxLayout" name="waveformSelectLayout">
          <item>
           <widget class="QComboBox" name="conditionOptions">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <item>
             <property name="text">
              <string>Sine Wave</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Square Wave</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectWaveform">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
